components/humantask/org.wso2.carbon.humantask/src/main/java/org/wso2/carbon/humantask/core/engine/runtime/xpath/JaxpFunctionResolver.java

```diff
@@ -194,18 +245,24 @@ public Object evaluate(List args) throws XPathFunctionException {
     /**
      * Returns the potential owners of the task. It MUST evaluate to an empty htt:organizationalEntity in case of an
      * error. If the task name is not present the current task MUST be considered.
+     * In : task name (optional)
+     * Out : potential owners (htt:organizationalEntity)
      */
     public class GetPotentialOwners implements XPathFunction {
+
         public Object evaluate(List args) throws XPathFunctionException {
             GenericHumanRoleDAO potentialOwners = null;
+            TaskDAO taskDAO = evalCtx.getTask();
```

Comment:
Yes it probably can't be null logically as of now, but your code shouldn't be vulnerable (break with a run time exception) if someone sends null context. So, it's always better to handle null checks or propagate appropriate exceptions and avoid possibility of run time exceptions.